Source/Controllers/Other/SQLitePinnedTableManager.swift

```diff
@@ -130,6 +131,10 @@ import OSLog
     }
 
     @objc func pinTable(hostName: String, databaseName: String, tableToPin: String) {
+        
+        if (pinnedTablesDatabaseDictionary[hostName]?[databaseName] != nil && pinnedTablesDatabaseDictionary[hostName]![databaseName]!.contains(tableToPin)) {
```

Comment:
`if let database = pinnedTablesDatabaseDictionary[hostName]?[databaseName], database.contains(tableToPin)`

Let's try to avoid force unwrap